### PR TITLE
[DOC] Updates to quickstart

### PR DESCRIPTION
**Pull request by PaulRMellor**
Signed-off-by: prmellor <pmellor@redhat.com>

**Documentation**

Minor updates to the quickstart guide following an editorial review.
Files also categorised in quickstart directories.

### Checklist

_Please go through this checklist and make sure all applicable tasks have been done_

- [ ] Update/write design documentation in `./design`
- [ ] Write tests
- [ ] Make sure all tests pass
- [x] Update documentation
- [ ] Check RBAC rights for Kubernetes / OpenShift roles
- [ ] Try your changes from Pod inside your Kubernetes and OpenShift cluster, not just locally
- [ ] Reference relevant issue(s) and close them after merging
- [ ] Update CHANGELOG.md

### CODE REVIEW

File: documentation/modules/quickstart/proc-downloading-product.adoc [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+
+.Procedure
+
+. Download the `strimzi-x.y.z.zip` file from {ReleaseDownload}.
+
+. Unzip the file to any destination.
++
+--
+* Windows or Mac: Extract the contents of the ZIP archive by double clicking on the ZIP file.
+* Red Hat Enterprise Linux: Open a terminal window in the target machine and navigate to where the ZIP file was downloaded.
```
Comment (ppatierno) [inline documentation/modules/quickstart/proc-downloading-product.adoc]: Why specific RHEL here? We should mention a more general "Linux"

File: documentation/modules/quickstart/proc-downloading-product.adoc [hunk 2]
```diff
@@ -0,0 +1,26 @@
+// Module included in the following assemblies:
+//
+// assembly-evaluation.adoc
+
+[id='proc-product-downloads-{context}']
+= Downloading {ProductName}
+
+A ZIP file contains the resources required for installation of {ProductName}, along with examples for configuration.
```
Comment (laidan6000) [inline documentation/modules/quickstart/proc-downloading-product.adoc]: "A ZIP file contains..." sounds a little generic to me, and I'm not sure about . You could reword to:

"The resources and artifacts required to install {ProductName}, along with examples for configuration, are provided in a ZIP file."

File: documentation/modules/quickstart/proc-downloading-product.adoc [hunk 3]
```diff
@@ -0,0 +1,26 @@
...
+
+.Procedure
+
+. Download the `strimzi-x.y.z.zip` file from {ReleaseDownload}.
+
+. Unzip the file to any destination.
++
+--
+* Windows or Mac: Extract the contents of the ZIP archive by double clicking on the ZIP file.
```
Comment (laidan6000) [inline documentation/modules/quickstart/proc-downloading-product.adoc]:
```suggestion
* Windows or Mac: Extract the contents of the ZIP archive by double-clicking on the ZIP file.
```

File: documentation/modules/quickstart/proc-sending-messages.adoc [hunk 4]
```diff
@@ -35,24 +35,22 @@ kubectl get service my-cluster-kafka-external-bootstrap -n my-kafka-project -o=j
...
-. Open a terminal and start the Kafka console producer with the topic `my-topic`:
+. Open a terminal, and start the Kafka console producer with the topic `my-topic`:
 +
 [source,shell,subs=+quotes]
 ----
 bin/kafka-console-producer.sh --broker-list _<node-address>_:_<node-port>_ --topic my-topic
 ----
 
-. Type your message into the console where the producer is running.
+. Type your message into the console where the producer is running, and press *Enter* to send the message.
```
Comment (laidan6000) [inline documentation/modules/quickstart/proc-sending-messages.adoc]: Enter a message

I think you could safely omit the second clause.

File: documentation/shared/snip-intro-operators.adoc [hunk 5]
```diff
@@ -0,0 +1,4 @@
+//standard operator intro text
```
Comment (laidan6000) [inline documentation/shared/snip-intro-operators.adoc]: Great snippet.

File: documentation/shared/snip-intro-operators.adoc [hunk 6]
```diff
@@ -0,0 +1,4 @@
...
+Operators are a method of packaging, deploying, and managing a Kubernetes application.
+{ProductName} Operators extend Kubernetes functionality, automating common and complex tasks related to a Kafka deployment.
```
Comment (laidan6000) [inline documentation/shared/snip-intro-operators.adoc]: Could give an example.

File: documentation/modules/quickstart/proc-downloading-product.adoc (hunk 2 above)
Comment (PaulRMellor) [reply documentation/modules/quickstart/proc-downloading-product.adoc]: Better :thumbsup:

File: documentation/shared/snip-intro-operators.adoc (hunk 6 above)
Comment (PaulRMellor) [reply documentation/shared/snip-intro-operators.adoc]: Not sure this snippet is the right place, as we may use it in more than one situation.

**Pull request merged**